src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2583,28 +2583,28 @@
     <value>Rotate selected {0, plural, one {image} other {images}} to the right</value>
   </data>
   <data name="OpenSettingsDescription" xml:space="preserve">
-    <value>Open settings page</value>
+    <value>Open the application settings page</value>
```

Comment:
Is the term 'application' helpful? Perhaps 'Open the settings page' would be better.

Reply:
I'd like to check that, do you remember where it is mentioned?